src/prepack-cli.js

```diff
@@ -454,6 +457,90 @@ fi
     return fatalErrors === 0;
   }
 
+  function generateDebugRepro(
+    sourceMaps: Array<string>,
+    originalSourceFiles: Array<{ absolute: string, relative: string }>
+  ) {
+    if (reproFilePath === undefined) return;
+
+    let reproZip = zipFactory();
+    // Copy all original sourcefiles used while Prepacking.
+    for (let file of originalSourceFiles) {
+      try {
+        // To avoid copying the "/User/name/..." version of the bundle/map/model included in originalSourceFiles
+        if (!reproFileNames.includes(file.relative)) {
+          console.log(`Zipping ${file.absolute}`);
+          let content = fs.readFileSync(file.absolute, "utf8");
+          reproZip.file(file.relative, content);
+        }
+      } catch (err) {
+        console.error(`Could not zip source file: ${err}`);
```

Comment:
Hm, this message might be easily ignored while leaving behind a corrupt repro zip and terminating with a successful error code. `generateDebugRepro` should return a Boolean, and overall Prepack should fail when `generateDebugRepro` fails.

Reply:
Different files not found can lead to different levels of corruption -- a missing input file vs. a missing source file (which may not even get opened). Perhaps it should error if an input file, prepack, or sourcemap can't be found, but let it go if a source file isn't there?

Reply:
Okay